app/main.py

```diff
@@ -1,8 +1,53 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
```

Comment:
bad formatting here
The braces must be located in one of two options: open and start with the text or have a line break between the text
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md